codec_specs.md

```diff
@@ -14,44 +14,99 @@ Each of these elements contain the same kind of data. And these data depend on t
-| V_REAL/???? | Real Video(TM) | The stream is one of the Real Video(TM) video streams listed below. Source for the codec names are from [Karl Lillevold on Doom9](http://forum.doom9.org/showthread.php?s=&amp;threadid=55773&amp;perpage=20&amp;pagenumber=2#post331855). The CodecPrivate element contains a "real_video_props_t" structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h). |
-
-| V_REAL/RV10 | RealVideo 1.0 aka RealVideo 5
- | Individual slices from the Real container are combined into a single frame. |
-| V_REAL/RV20 | RealVideo G2 and RealVideo G2+SVT | Individual slices from the Real container are combined into a single frame. |
-| V_REAL/RV30 | RealVideo 8 | Individual slices from the Real container are combined into a single frame. |
-| V_REAL/RV40 | rv40 : RealVideo 9 | Individual slices from the Real container are combined into a single frame. |
-
- |
-| V_QUICKTIME | Video taken from QuickTime(TM) files | Several codecs as stored in QuickTime, e.g. Sorenson or Cinepak. The CodecPrivate contains all additional data that is stored in the 'stsd' (sample description) atom in the QuickTime file **after** the mandatory video descriptor structure (starting with the size and FourCC fields). For an explanation of the QuickTime file format read [QuickTime File Format Specification](https://developer.apple.com/library/mac/documentation/QuickTime/QTFF/QTFFPreface/qtffPreface.html). |
-
-| V_THEORA | Theora | The private data contains the first three Theora packets in order. The lengths of the packets precedes them. The actual layout is: 
+The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (matroska-devel at lists.matroska.org).
+
+
+## Video Codecs
+
+  
+**Codec ID:** "V_MS/VFW/FOURCC"  
+**Codec Name:** Microsoft (TM) Video Codec Manager (VCM)  
+**Description:** The private data contains the VCM structure BITMAPINFOHEADER including the extra private bytes, as [defined by Microsoft](http://msdn.microsoft.com/library/default.asp?url=/library/en-us/gdi/bitmaps_1rw2.asp). The data are stored in little endian format (like on IA32 machines). Where is the Huffman table stored in HuffYUV, not AVISTREAMINFO ??? And the FourCC, not in AVISTREAMINFO.fccHandler ???
+
+  
+**Codec ID:** V_UNCOMPRESSED  
+**Codec Name:** Video, raw uncompressed video frames  
+**Description:** The private data is void, all details about the used colour specs and bit depth are to be put/read from the KaxCodecColourSpace elements.
+
+  
+**Codec ID:** V_MPEG4/ISO/???  
```

Comment:
This `???` Codec ID and the Real/??? one below aren't real Codec IDs, but just like a Header for the Codec IDs that follow them. I think they could be removed.